src/containers/App/assets/app.scss

```diff
@@ -27,14 +27,14 @@
 
 .title {
   color: $color-white;
-  font-size: 5em;
+  font-size: $font-xxxl;
   margin: 0;
   text-align: center;
 }
 
 .tagline {
   color: $color-white;
-  font-size: 2em;
+  font-size: $font-xl;
   font-weight: 100;
   margin: 10px 0px;
```

Comment:
Please be consistent with margins. You've defined them with `rem` above. Stick with one unit.

Reply:
@emmawedekind are we going to have consistent white space in the designs? If so we definitely would have to create some constants or common css classes to avoid this kind of issues.